core/shared/src/main/scala/spire/math/Polynomial.scala

```diff
@@ -490,14 +485,16 @@ with EuclideanRing[Polynomial[C]] with VectorSpace[Polynomial[C], C] {
   override def quotmod(x: Polynomial[C], y: Polynomial[C]): (Polynomial[C], Polynomial[C]) = x /% y
 
   final def gcd(x: Polynomial[C], y: Polynomial[C]): Polynomial[C] = {
-    val result = euclid(x, y)(Polynomial.eq)
+    val result = euclid(x, y)(Polynomial.eq, this)
     if (result.degree > 0) {
       result
     } else {
       // return the gcd of all coefficients when there is no higher degree divisor
       Polynomial.constant(spire.math.gcd(x.coeffsArray ++ y.coeffsArray))
     }
   }
+
+  def lcm(x: Polynomial[C], y: Polynomial[C]): Polynomial[C] = (x /~ gcd(x, y)) * y //FIXME?
```

Comment:
I think this is fine - we know that the remainder will be 0, so using `/~` is legit.